project-1/Asgun-alt/pkg/domain/books/models.go

```diff
@@ -6,35 +6,36 @@ import (
 	"gorm.io/gorm"
 )
 
-type BookDomain struct {
-	BookID      string         `gorm:"primaryKey, column:book_id"`
-	Title       string         `gorm:"column:title"`
-	Author      string         `gorm:"column:author"`
-	Publisher   string         `gorm:"column:publisher"`
-	BookSummary string         `gorm:"column:book_summary"`
-	Stock       int            `gorm:"column:stock"`
-	MaxStock    int            `gorm:"column:max_stock"`
-	Created_At  time.Time      `gorm:"column:created_at"`
-	Updated_At  time.Time      `gorm:"column:updated_at"`
-	Deleted_At  gorm.DeletedAt `gorm:"column:deleted_at"`
+type BookCollections struct {
+	BookID      uint           `gorm:"autoIncrement:true, primaryKey, column:book_id" json:"book_id"`
```

Comment:
di penulisan tag, pemisahannya pakai `;`, bukan `, `
```suggestion
	BookID      uint           `gorm:"autoIncrement:true;primaryKey;column:book_id" json:"book_id"`
```